db/migrations/20200831145752_creat_foo_bars.cr

```diff
@@ -0,0 +1,22 @@
+class CreatFooBars::V20200831145752 < Avram::Migrator::Migration::V1
```

Comment:
I have no idea what this is about. I think I just started something and didn't finish